src/kibana/components/courier/saved_object/saved_object.js

```diff
@@ -233,7 +233,7 @@ define(function (require) {
           if (_.deepGet(err, 'origError.status') === 409 && window.confirm(confirmMessage)) {
             return docSource.doIndex(source).then(finish);
           }
-          return Promise.resolve(false);
+          return Promise.reject(err.message);
```

Comment:
So I think this should be rejecting with the err object and not the err.message. If the notifier needs a string then we should add a test to the notifier to see if it's an instance of Error and use err.message.